Source/Blazorise/wwwroot/popper.js

```diff
@@ -0,0 +1,37 @@
+
+const DIRECTION_DEFAULT = 'Default'
+const DIRECTION_DOWN = 'Down'
+const DIRECTION_UP = 'Up'
+const DIRECTION_END = 'End'
+const DIRECTION_START = 'Start'
+
+function getPopperDirection(direction)
+{
+    if (direction == DIRECTION_DEFAULT || direction == DIRECTION_DOWN)
+        return "bottom-start";
+    if (direction == DIRECTION_UP)
+        return "top-start";
+    if (direction == DIRECTION_END)
+        return "right-start";
+    if (direction == DIRECTION_START)
+        return "left-start";
+    return "bottom-start";
+}
+
+export function createPopper(elementOffset, menu, options) {
```

Comment:
Are we gonna be able to import a popper module like this for the `tooltip.js`?

Reply:
I actually didn't see how tooltip. Js is done but I don't see why not

Reply:
So for tooltip.js I see a tippy call 
I can't tell, aren't those two distinct libraries?

Reply:
I suspect that tippy is expecting a popper not as a module but as a regular JS script. I think the only way to make it work is to change the tippy code to initialize popper using our own exported `createPopper` method, somehow.